package.json

```diff
@@ -26,7 +26,7 @@
     "axios": "^0.18.0",
     "dotenv": "^6.1.0",
     "express": "^4.16.4",
-    "ipfs": "^0.32.3",
+    "ipfs": "^0.33.1",
```

Comment:
@zachferland Are we fine with updating the ipfs version?

Reply:
Looking at releases it looks like it shouldn't break anything: https://github.com/ipfs/js-ipfs/releases

Reply:
Maybe fine here, but kept 32 in libs since that is the ipfs post message proxy uses. Shouldn't be any issues using different versions in each, probably a good idea to update where we can. I think fine if we test a bit.

Reply:
I'll do some testing. Thanks @zachferland